jumpscale/packages/tfgrid_solutions/chats/exposed.py

```diff
@@ -12,6 +12,7 @@
     "ubuntu": solutions.list_ubuntu_solutions,
     "flist": solutions.list_flist_solutions,
     "gitea": solutions.list_gitea_solutions,
+    "blog": solutions.list_blog_solutions,
```

Comment:
cannot we have a generic list_solutions or something like that that return this list with possible kinds or something? because if someone added a solution and didn't add its kind here, then it would be broken?